en/components/grid_toolbar.md

```diff
@@ -135,8 +139,15 @@ The Grid Toolbar service has a few more APIs to explore, which are listed below.
 The following inputs are available on the **IgxGrid** component:
 | Name | Type | Description |
 | :--- | :--- | :--- |
+| `toolbar` | IgxGridToolbarComponent | A reference to the toolbar component. |
 | `showToolbar` | boolean | Show or hide the grid's toolbar. |
 | `toolbarTitle` | string | Sets the toolbar title. |
+| `columnHiding` | boolean | Show or hide the column hiding feature. |
```

Comment:
I suggest we order these as follows:
- columnHiding
- columnHidingTitle
- hiddenColumnsText 
Same for Pinning

Reply:
Done